configure.ac

```diff
@@ -38,6 +36,20 @@ else
         AC_MSG_WARN([C11 stdatomic support unavailable. Falling back to slow mutex])
 fi
 
+dir=""
+AC_ARG_WITH([socket-path], AS_HELP_STRING([--with-socket-path=SOCKET_PATH],
+            [path to create unix socket @<:@default=/run/clr-debug-info@:>@]), [dir=${withval}],
+            [dir="/run/clr-debug-info"])
+test -z "${dir}" && dir=/run/clr-debug-info
+AC_DEFINE_UNQUOTED([SOCKET_PATH], ["${dir}"], [path to create unix socket @<:@default=/run/clr-debug-info@:>@])
+
+dir=""
+AC_ARG_WITH([cache-dir], AS_HELP_STRING([--with-cache-dir=CACHE_DIR],
+            [path to cache downloaded content @<:@default=/var/cache/debuginfo@:>@]), [dir="${withval}"],
+            [dir="/var/cache/debuginfo"])
+test -z "${dir}" && dir=/var/cache/debuginfo
+AC_DEFINE_UNQUOTED([CACHE_DIR], ["${dir}"], [path to cache downloaded content @<:@default=/var/cache/debuginfo@:>@])
```

Comment:
Can you also update the `AC_MSG_RESULT` macro below to print the values for the two paths? You will want to use different shell variables to store the paths (instead of reusing `dir`) so that the values can be printed separately.